pkg/plugin/loader/loader.go

```diff
@@ -73,7 +73,11 @@ func LoadAllPlugins(namespace, sonobuoyImage, imagePullPolicy string, searchPath
 		pluginDefinitions = append(pluginDefinitions, pluginDefinition)
 	}
 
-	pluginDefinitions = filterPluginDef(pluginDefinitions, selections)
+	// The zero value for the slice, nil, indicates that all plugins should be run.
+	// If the user wants to run 0 plugins, they should explicitly pass an empty array.
```

Comment:
What's the use-case to not run any plugins? Should that be an error?

Reply:
So currently the use case for that, not that I'm sure anyone uses it, is that you want to do either:
 - run/test sonobuoy works at all
 - run sonobuoy to just get the query data

So the reason I want to continue to allow that is more about limiting truly breaking changes when they aren't really necessary. I agree it is not a common flow, but I don't really see a reason to not allow it if someone wants to explicitly request it.